src/common/components/tableDropdown/TableDropdown.tsx

```diff
@@ -0,0 +1,192 @@
+import classNames from 'classnames';
+import { IconAngleDown } from 'hds-react';
+import React, { ReactElement } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import useKeyboardNavigation from '../../../hooks/useDropdownKeyboardNavigation';
+import styles from './tableDropdown.module.scss';
+
+export type MenuItemProps = {
+  children: ReactElement;
+  onClick: (row: Record<string, unknown>) => void;
+};
+
+type InternalMenuItemProps = MenuItemProps & {
+  isFocused: boolean;
+  row: Record<string, unknown>;
+  toggleMenu: () => void;
+};
+
+const MenuItem: React.FC<InternalMenuItemProps> = ({
+  children,
+  isFocused,
+  onClick,
+  row,
+  toggleMenu,
+}) => {
+  const component = React.useRef<HTMLLIElement>(null);
+
+  const handleClick = () => {
+    onClick(row);
+    toggleMenu();
+  };
+
+  const onKeyDown = (event: React.KeyboardEvent<HTMLLIElement>) => {
+    if (event.key === 'Enter') {
+      handleClick();
+    }
+  };
+
+  React.useEffect(() => {
+    if (isFocused) {
+      component.current?.focus();
+    }
+  }, [isFocused]);
+
+  return (
+    <li
+      ref={component}
+      className={classNames({
+        [styles.isFocused]: isFocused,
+      })}
+      onClick={handleClick}
+      onKeyDown={onKeyDown}
+      role="menuitem"
+      tabIndex={0}
+    >
+      {children}
+    </li>
+  );
+};
+
+interface Props {
+  items: MenuItemProps[];
+  row: Record<string, unknown>;
+}
+
+const ActionsDropdown: React.FC<Props> = ({ row, items }) => {
```

Comment:
Should this be named TableDropdown?

Reply:
Yes, absolutely. Forgot to change when copy-pasted code from previous place